tmail-backend/tmail-third-party/ai-bot/src/test/java/com/linagora/tmail/mailet/AIRedactionalHelperTest.java

```diff
@@ -40,10 +40,10 @@ public class AIRedactionalHelperTest {
     @BeforeEach
     void setUp() throws Exception{
         aiBotConfig= new AIBotConfig(
-            "demo",
+            "sk-myTzTkVkqhRCp2BEVsyvjA",
```

Comment:
Is this a real or a fake key?

If fake please:

```suggestion
            "sk-fakefakefakefakefakefakefakefake",
```